src/seedu/addressbook/parser/Parser.java

```diff
@@ -96,7 +96,7 @@ public Command parseCommand(String userInput) {
      * @param args full command args string
      * @return the prepared command
      */
-    private Command prepareAdd(String args){
+    private static Command prepareAdd(String args){
```

Comment:
You can use a class level method when you don't necessarily need an instance of the object. Like StringUtils or Parser.